### ISSUE TITLE
SyntaxError: Unexpected end of JSON input

### ISSUE THREAD

**Issue by unolx** [diciaup/assistant-cli#4]
**Bug**
The following message appears for each query:

```
processing... |SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Socket.<anonymous> (C:\Users\username\AppData\Roaming\npm\node_modules\assistant-cli\dist\core.js:34:48)
    at Socket.emit (node:events:513:28)
    at addChunk (node:internal/streams/readable:324:12)
    at readableAddChunk (node:internal/streams/readable:297:9)
    at Readable.push (node:internal/streams/readable:234:10)
    at Pipe.onStreamRead (node:internal/stream_base_commons:190:23) error:

```

The response, however, is returned and displayed correctly.

**AssistantCli Version**
1.3.4

**To Reproduce**
1. Launch command 'assistant Hello World'
2. See error
3. Try to launch assistant clean
4. Re-launch command 'assistant Hello World'
5. See error

**CLI:**
 - OS: Win
 - OS Version 11
 - AssistantCli Version 1.3.4

**Comment by diciaup** [comment]
Solved on version 1.3.5

**Comment by unolx** [comment]
Thanks works wonderfully now.
